src/ascent/runtimes/flow_filters/ascent_runtime_rendering_filters.cpp

```diff
@@ -499,34 +505,101 @@ class CinemaManager
   std::string                          m_csv;
 
   vtkm::Bounds                         m_bounds;
-  const int                            m_phi;
-  const int                            m_theta;
+  int                                  m_phi;
+  float                                m_phi_min;
+  float                                m_phi_inc;
+  int                                  m_theta;
+  float                                m_theta_min;
+  float                                m_theta_inc;
   std::string                          m_image_name;
   std::string                          m_image_path;
   std::string                          m_db_path;
   std::string                          m_base_path;
   float                                m_time;
 public:
   CinemaManager(vtkm::Bounds bounds,
-                const int phi,
-                const int theta,
+                const conduit::Node &render_node,
                 const std::string image_name,
                 const std::string path)
     : m_bounds(bounds),
-      m_phi(phi),
-      m_theta(theta),
       m_image_name(image_name),
       m_time(0.f)
   {
+    // Handle phi.
+    if(render_node.has_path("phi"))
+    {
+      m_phi_min = -180.0;
+      m_phi = render_node["phi"].as_int32();
```

Comment:
small suggested change - in the places were you have `as_int32` you can use `to_int32` to avoid an error if folks passed an int64 (or other int type)

Reply:
Changed.